lib/rb_shift/util/variable_resolver.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module RbShift
+  module Util
+    # Supporting module to create instance variables of OpenshiftKind
+    module VariableResolver
+      def get_instance_variable(name:, type:, reload: false, filter: nil)
```

Comment:
I'm still kinda not sure about this method as it seems a bit cryptic since it adds a bunch of reflection magic to quite simple task really.

Reply:
Due to nature of modification, which was performed on several classes it is in my opinion better to write a module that contains the same functionality, but as you've pointed out, this PR is irrelevant since the objects will be stored in hashes.